reflex-dom-core/src/Reflex/Dom/Main.hs

```diff
@@ -15,6 +15,8 @@ module Reflex.Dom.Main where
 
 import Prelude hiding (concat, mapM, mapM_, sequence, sequence_)
 
+import Reflex.Class (holdDyn)
+import Reflex.Dom.Builder.Class (MountState(Mounted, Mounting))
```

Comment:
The whitespace style is just slightly different here.  Running `stylish-haskell` from the root of this project should work - but in this particular case, it's just that `MountState` should get a space after it.